tutorials/sinon/stub.md

```diff
@@ -0,0 +1,14 @@
+With the `stub()` function, you can make testing difficult parts of your code easy.
+`stub()` substitutes the real function and returns a value you can define using `callsfake()`.
+All stubs are spies in sinon.js, therefore, you can access the `callCount` property and see how
```

Comment:
Doesn't really make sense to say that all stubs are spies, since we haven't introduced spies yet :) Just indicate that stubs have a `callCount` property, as well as `calledOnce` and `getCall()`, that you can use to check whether the function has been called in tests.